packages/oasis-actions/src/protocols/aave/getAaveProtocolData.ts

```diff
@@ -28,29 +33,29 @@ export const getAaveProtocolData = async ({
   provider,
   protocolVersion,
 }: AaveProtocolDataArgs) => {
-  const isV2 = protocolVersion === 2
-  const isV3 = protocolVersion === 3
+  const isV2 = protocolVersion === AaveVersion.v2
+  const isV3 = protocolVersion === AaveVersion.v3
   const hasProxy = !!proxy
 
   let priceOracle
   let aavePool
-  if (isV2 && 'priceOracle' in addresses) {
+  if (isV2 && aaveV2UniqueContractName in addresses) {
```

Comment:
For me, it's not a very efficient way to check if the current object has given property. 
I had some issues with that. I created a helper function in frontend: 
```
getAddressesFromContext(
  context: Context,
): AAVEStrategyAddresses & AAVEV3StrategyAddresses
```
So I can prepare addresses with one function, and I don't care about the structure of that object. I only want to satisfy the interface. 
As a client, I can have a `pool` address in V2 addresses.

Reply:
Fab. Merged your PR